packages/client/src/app/pages/portfolio/portfolio.component.html

```diff
@@ -1,62 +1,122 @@
+                    <span class="icon"><i-feather name="feather"></i-feather></span>
+                    <span class="stat">{{ portUser.stats.works | abbreviate }}</span>
                 </div>
-                <div class="port-nav">
-                    <span>
-                        <a class="port-nav-button button-left" [routerLink]="['/portfolio', portUser._id, portUser.username | slugify, 'blog']" routerLinkActive="router-link-active">{{ portUser.stats.blogs }} Blog{{ portUser.stats.blogs | pluralize }}</a>
-                        <a class="port-nav-button" [routerLink]="['/portfolio', portUser._id, portUser.username | slugify, 'works']" routerLinkActive="router-link-active">{{ portUser.stats.works }} Work{{ portUser.stats.works | pluralize }}</a>
-                        <a class="port-nav-button" [routerLink]="['/portfolio', portUser._id, portUser.username | slugify, 'collections']" routerLinkActive="router-link-active">Collections</a>
-                    </span>
+                <div class="blogs" matTooltip="Blogs" matTooltipClass="offprint-tooltip">
+                    <span class="icon"><i-feather name="coffee"></i-feather></span>
+                    <span class="stat">{{ portUser.stats.blogs | abbreviate }}</span>
                 </div>
-                <div class="stats">
-                    <p>
-                        <a>{{ portUser.stats.watchers }} watcher{{ portUser.stats.watchers | pluralize }}</a> // 
-                        <a>{{ portUser.stats.watching }} watching</a>
-                    </p>
+                <div class="watchers" matTooltip="Watchers" matTooltipClass="offprint-tooltip">
+                    <span class="icon"><i-feather name="eye"></i-feather></span>
+                    <span class="stat">{{ portUser.stats.watchers | abbreviate }}</span>
                 </div>
             </div>
         </div>
+        <div class="port-controls" *ngIf="currentUser && !currentUserIsSame()">
+            <button class="control left" matTooltip="Start a Conversation" matTooltipClass="offprint-tooltip"><i-feather name="message-circle"></i-feather></button>
+            <button class="control mid" matTooltip="Send Friend Request" matTooltipClass="offprint-tooltip"><i-feather name="user-plus"></i-feather></button>
+            <button class="control mid" matTooltip="Watch" matTooltipClass="offprint-tooltip"><i-feather name="eye"></i-feather></button>
+            <button class="control right" matTooltip="Report" matTooltipClass="offprint-tooltip"><i-feather name="flag"></i-feather></button>
+        </div>
+        <div class="port-nav">
+            <ul class="navigation-list">
+                <li>
+                    <a [routerLink]="['home']" routerLinkActive="router-link-active">
+                        <span class="link-icon">
+                            <i-feather name="home"></i-feather>
+                        </span>
+                        Home
```

Comment:
For the future, I recommend using strings from the Constants file instead of writing them out each time.